### ISSUE TITLE
[feature request] Support select entity from Home Assistant

### ISSUE THREAD

**Issue by mvdwetering** [unfoldedcircle/bug-reports#211]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Home Assistant has select entities, some integrations expose features as select entiteis which would be useful to have on the remote. 

In my case it are the select entities for the Hue Sync Box integration. It has select entities to select the mode game/video/music and intensity subtle/normal/high/extreme.  Other people might have other useful selects.

I would kind of expect it to behave like the source selection on mediaplayers. So you can assign buttons to a specific option in the select, but I would also want to be able to cycle through the options with 1 button by pressing it repeatedly (next service in HA).

**Comment by zehnm** [comment]
@mvdwetering I constantly learn new things in Home Assistant, select entities for example :-)

Are you using this custom Hue Sync Box integration <https://github.com/mvdwetering/huesyncbox>?
I have a Sync Box as well and love it. But so far I just have it integrated through the bridge and the default HA Philips Hue integration. Mostly in auto-mode for watching movies. No time for gaming...

I'll dig into select entities and a way on how to integrate it.

**Comment by mvdwetering** [comment]
Yeah, I am using that integration.  I should probably have also mentioned that I am the author of that integration 😓

**Comment by zehnm** [comment]
If I'd better paid attention to your username I should have noticed :-)
So far I've managed to install HACS and the Hue Sync Box integration. Now I need to find time to look into the select entities.

**Comment by RC-Thoughts** [comment]
@zehnm the Logitech Harmony in HA-integration is also in need of this, the switches we use now to start activities will be depricated.

**Comment by klada** [comment]
@zehnm When looking at this please also take `input_select` entities into account:

https://www.home-assistant.io/integrations/input_select/

For example I have a `input_select.tv_picture_mode` which allows me to set the picture mode (Filmmaker mode, Dolby Cinema, Game Mode, ISF Expert, ...). These should be exposed to the remote as one entity which allows to set a proper mode from a drop-down or pop up.

**Comment by Kamblian** [comment]
Is related to https://github.com/unfoldedcircle/feature-and-bug-tracker/issues/280 [feature request] Add "Helpers" to Home Assistant Integration

**Comment by zehnm** [comment]
Select-entities will be supported in the next beta release!
- https://unfoldedcircle.github.io/core-api/entities/entity_select.html
- https://github.com/unfoldedcircle/integration-home-assistant/pull/83

<img width="377" height="378" alt="Image" src="https://github.com/user-attachments/assets/08fde612-3047-4f7e-b69f-242214e5b286" />

**Comment by zehnm** [comment]
Remote firmare 2.8.3 supports select-entities

**Comment by mvdwetering** [comment]
Will this still become available for the Rremote 2?
I haven't seen firmware updates in over a year now 😞

**Comment by JackJPowell** [comment]
It's here today! Just enable beta updates and you'll be set. Nearly everyone is running from the beta channel. It is very stable and has tons of new features if you are still running 1.7.3. 

I think you'll need to update twice. Once to 2.6.13ish than another to 2.9.1 

Enjoy

**Comment by zehnm** [comment]
You can check the changelog for new features on the webpage and filter by Remote model: https://unfoldedcircle.com/changelog
or on Discord
- Releases: https://discord.com/channels/553671366411288576/1143871904038531185
- Beta: https://discord.com/channels/553671366411288576/1253076993566572706

Remote Two received the 2.8.5 update on Februrary 27 in the beta channel.
We are well aware of the too long beta state. There are still a few things to iron out for a release and going back to shorter beta cycles.

**Comment by mvdwetering** [comment]
The time that I use the remote is my relax time I really don't want to run/debug beta software during that time.  Even if it works for other people apparently there is still something blocking it from being stable. Glad to hear it is still coming, I will just wait.
